src/cli/Clip.cpp

```diff
@@ -39,12 +40,18 @@ const QCommandLineOption Clip::TotpOption =
                                      << "totp",
                        QObject::tr("Copy the current TOTP to the clipboard (equivalent to \"-a totp\")."));
 
+const QCommandLineOption Clip::BestMatchOption = QCommandLineOption(
+    QStringList() << "b"
+                  << "best-match",
+    QObject::tr("try to find the unique entry matching, will fail and display the list of matches otherwise."));
```

Comment:
```suggestion
    QObject::tr("Try to find the unique entry matching, will fail and display the list of matches otherwise."));
```
for consistency?

Reply:
indeed. Fixed